pymedphys/_gui/streamlit/mudensity/main.py

```diff
@@ -18,36 +18,31 @@
 # pylint: disable = too-many-lines, redefined-outer-name
```

Comment:
can you get rid of this pylint disablement now (or has this already happened in the future?)

Reply:
Yup, was able to get rid of it :). Happened in the PR on top of this one:

https://github.com/pymedphys/pymedphys/pull/1125/files